### PR TITLE
resets the array of layers if not passed

### PR DESCRIPTION
**Pull request by abdullaBest**

### CODE REVIEW

**Comment by dmliao** [comment]
Hi there! Thanks for submitting a pull request. I'd appreciate some more context over what is the intention here, though; is it so that when `XRRenderStateInit.layers` doesn't exist, the `layers` field of the XRSession should be undefined? (VS an empty array, as currently written).

**Comment by abdullaBest** [comment]
it will not be undefined, it will remain empty. This edit is needed so that when calling updateRenderState without the layers parameter, for example (inside threejs ):
```
session.updateRenderState( {
    depthNear: cameraVR.near,
    depthFar: cameraVR.far
});
```
layers value was not reset

File: src/api/XRSessionWithLayer.ts [hunk 1]
```diff
@@ -288,7 +288,9 @@ export class XRSessionWithLayer {
 	public updateRenderState(XRRenderStateInit: XRRenderStateWithLayer): void {
 		this.existingBaseLayer = XRRenderStateInit.baseLayer
 
-		this.layers = XRRenderStateInit.layers || []
+		if (XRRenderStateInit.layers){
```
Comment (dmliao) [inline src/api/XRSessionWithLayer.ts]: nit: Can you run `npm run format` on the pull request before merging?

**Review by dmliao — COMMENTED** [review]
Can you run `npm run format` on your pull request before we merge?

File: src/api/XRSessionWithLayer.ts (hunk 1 above)
Comment (abdullaBest) [reply src/api/XRSessionWithLayer.ts]: done

**Pull request merged**